panel/src/components/Navigation/Search.vue

```diff
@@ -150,6 +150,9 @@ export default {
       };
     },
     navigate(item) {
+
+      console.log(item);
```

Comment:
This shouldn’t be here :)

Reply:
Argh.

Reply:
Sometimes I wish there was a "stupidity check" before you can push